### PR TITLE
Update de* languages

### PR DESCRIPTION
**Pull request by zero-24**

### CODE REVIEW

File: joomla_v4/translations/core/installation/language/de-AT/langmetadata.xml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <metafile client="installation" version="4.0">
 	<name>German (Austria)</name>
-	<version>4.0.0</version>
-	<creationDate>11.08.2021</creationDate>
+	<version>4.0.3v1</version>
```
Comment (conconnl) [inline joomla_v4/translations/core/installation/language/de-AT/langmetadata.xml:4]: Should be 4.0.3
See: https://github.com/joomla/core-translations/blob/main/joomla_v4/source/installation/language/en-GB/langmetadata.xml#L4

File: joomla_v4/translations/core/installation/language/de-AT/langmetadata.xml [hunk 2]
```diff
@@ -1,8 +1,8 @@
...
+	<creationDate>14.09.2021</creationDate>
```
Comment (conconnl) [inline joomla_v4/translations/core/installation/language/de-AT/langmetadata.xml:5]: Should be September 2021
See: https://github.com/joomla/core-translations/blob/main/joomla_v4/source/installation/language/en-GB/langmetadata.xml#L5

File: joomla_v4/translations/core/installation/language/de-CH/langmetadata.xml [hunk 3]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <metafile client="installation" version="4.0">
 	<name>German (Switzerland)</name>
-	<version>4.0.0</version>
-	<creationDate>11.08.2021</creationDate>
+	<version>4.0.3v1</version>
```
Comment (conconnl) [inline joomla_v4/translations/core/installation/language/de-CH/langmetadata.xml:4]: Should be 4.0.3
See: https://github.com/joomla/core-translations/blob/main/joomla_v4/source/installation/language/en-GB/langmetadata.xml#L4

File: joomla_v4/translations/core/installation/language/de-DE/langmetadata.xml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <metafile client="installation" version="4.0">
 	<name>German (Germany)</name>
-	<version>4.0.0</version>
-	<creationDate>11.08.2021</creationDate>
+	<version>4.0.3v1</version>
```
Comment (conconnl) [inline joomla_v4/translations/core/installation/language/de-DE/langmetadata.xml:4]: Should be 4.0.3
See: https://github.com/joomla/core-translations/blob/main/joomla_v4/source/installation/language/en-GB/langmetadata.xml#L4

File: joomla_v4/translations/core/installation/language/de-LI/langmetadata.xml [hunk 5]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <metafile client="installation" version="4.0">
 	<name>German (Lichtenstein)</name>
-	<version>4.0.0</version>
-	<creationDate>11.08.2021</creationDate>
+	<version>4.0.3v1</version>
```
Comment (conconnl) [inline joomla_v4/translations/core/installation/language/de-LI/langmetadata.xml:4]: Should be 4.0.3
See: https://github.com/joomla/core-translations/blob/main/joomla_v4/source/installation/language/en-GB/langmetadata.xml#L4

File: joomla_v4/translations/core/installation/language/de-LU/langmetadata.xml [hunk 6]
```diff
@@ -1,8 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <metafile client="installation" version="4.0">
 	<name>German (Luxembourg)</name>
-	<version>4.0.0</version>
-	<creationDate>11.08.2021</creationDate>
+	<version>4.0.3v1</version>
```
Comment (conconnl) [inline joomla_v4/translations/core/installation/language/de-LU/langmetadata.xml:4]: Should be 4.0.3
See: https://github.com/joomla/core-translations/blob/main/joomla_v4/source/installation/language/en-GB/langmetadata.xml#L4

File: joomla_v4/translations/core/installation/language/de-CH/langmetadata.xml [hunk 7]
```diff
@@ -1,8 +1,8 @@
...
+	<creationDate>14.09.2021</creationDate>
```
Comment (conconnl) [inline joomla_v4/translations/core/installation/language/de-CH/langmetadata.xml:5]: Should be September 2021
See: https://github.com/joomla/core-translations/blob/main/joomla_v4/source/installation/language/en-GB/langmetadata.xml#L5

File: joomla_v4/translations/core/installation/language/de-DE/langmetadata.xml [hunk 8]
```diff
@@ -1,8 +1,8 @@
...
+	<creationDate>14.09.2021</creationDate>
```
Comment (conconnl) [inline joomla_v4/translations/core/installation/language/de-DE/langmetadata.xml:5]: Should be September 2021
See: https://github.com/joomla/core-translations/blob/main/joomla_v4/source/installation/language/en-GB/langmetadata.xml#L5

File: joomla_v4/translations/core/installation/language/de-LI/langmetadata.xml [hunk 9]
```diff
@@ -1,8 +1,8 @@
...
+	<creationDate>14.09.2021</creationDate>
```
Comment (conconnl) [inline joomla_v4/translations/core/installation/language/de-LI/langmetadata.xml:5]: Should be September 2021
See: https://github.com/joomla/core-translations/blob/main/joomla_v4/source/installation/language/en-GB/langmetadata.xml#L5

File: joomla_v4/translations/core/installation/language/de-LU/langmetadata.xml [hunk 10]
```diff
@@ -1,8 +1,8 @@
...
+	<creationDate>14.09.2021</creationDate>
```
Comment (conconnl) [inline joomla_v4/translations/core/installation/language/de-LU/langmetadata.xml:5]: Should be September 2021
See: https://github.com/joomla/core-translations/blob/main/joomla_v4/source/installation/language/en-GB/langmetadata.xml#L5

**Review by conconnl — CHANGES_REQUESTED** [review]

File: joomla_v4/translations/core/installation/language/de-AT/langmetadata.xml (hunk 1 above)
Comment (zero-24) [reply joomla_v4/translations/core/installation/language/de-AT/langmetadata.xml:4]:
```suggestion
	<version>4.0.3</version>
```

File: joomla_v4/translations/core/installation/language/de-AT/langmetadata.xml (hunk 2 above)
Comment (zero-24) [reply joomla_v4/translations/core/installation/language/de-AT/langmetadata.xml:5]:
```suggestion
	<creationDate>September 2021</creationDate>
```

File: joomla_v4/translations/core/installation/language/de-CH/langmetadata.xml (hunk 3 above)
Comment (zero-24) [reply joomla_v4/translations/core/installation/language/de-CH/langmetadata.xml:4]:
```suggestion
	<version>4.0.3</version>
```

File: joomla_v4/translations/core/installation/language/de-DE/langmetadata.xml (hunk 4 above)
Comment (zero-24) [reply joomla_v4/translations/core/installation/language/de-DE/langmetadata.xml:4]:
```suggestion
	<version>4.0.3</version>
```

File: joomla_v4/translations/core/installation/language/de-LI/langmetadata.xml (hunk 5 above)
Comment (zero-24) [reply joomla_v4/translations/core/installation/language/de-LI/langmetadata.xml:4]:
```suggestion
	<version>4.0.3</version>
```

File: joomla_v4/translations/core/installation/language/de-LU/langmetadata.xml (hunk 6 above)
Comment (zero-24) [reply joomla_v4/translations/core/installation/language/de-LU/langmetadata.xml:4]:
```suggestion
	<version>4.0.3</version>
```

File: joomla_v4/translations/core/installation/language/de-CH/langmetadata.xml (hunk 7 above)
Comment (zero-24) [reply joomla_v4/translations/core/installation/language/de-CH/langmetadata.xml:5]:
```suggestion
	<creationDate>September 2021</creationDate>
```

File: joomla_v4/translations/core/installation/language/de-DE/langmetadata.xml (hunk 8 above)
Comment (zero-24) [reply joomla_v4/translations/core/installation/language/de-DE/langmetadata.xml:5]:
```suggestion
	<creationDate>September 2021</creationDate>
```

File: joomla_v4/translations/core/installation/language/de-LI/langmetadata.xml (hunk 9 above)
Comment (zero-24) [reply joomla_v4/translations/core/installation/language/de-LI/langmetadata.xml:5]:
```suggestion
	<creationDate>September 2021</creationDate>
```

File: joomla_v4/translations/core/installation/language/de-LU/langmetadata.xml (hunk 10 above)
Comment (zero-24) [reply joomla_v4/translations/core/installation/language/de-LU/langmetadata.xml:5]:
```suggestion
	<creationDate>September 2021</creationDate>
```

**Review by conconnl — APPROVED** [review]

**Pull request merged**